src/docs/markdown/caddyfile/directives/handle_errors.md

```diff
@@ -60,6 +60,20 @@ handle_errors {
 }
 ```
 
+If you want to provide custom error pages only for some error codes, you can check the existence of the custom error files beforehand with a [`file`](/docs/caddyfile/matchers#file) matcher:
+
+```caddy-d
+handle_errors {
+	@custom_err file /err-{http.error.status_code}.html /err.html
+	handle @custom_err {
+		rewrite @custom_err {http.matchers.file.relative}
```

Comment:
I probably should have reviewed this first though, lol.

You can remove the @custom_err from `rewrite`

Reply:
Woops, sorry. :see_no_evil: